src/tty_interface.c

```diff
@@ -65,7 +65,11 @@ static void draw_match(tty_interface_t *state, const char *choice, int selected)
 		} else {
 			tty_setfg(tty, TTY_COLOR_NORMAL);
 		}
-		tty_printf(tty, "%c", choice[i]);
+		if (choice[i] == '\n') {
+			tty_putc(tty, ' ');
```

Comment:
What do you think about using `'⏎'` instead? I think it looks cooler but am not sure it's actually a benefit 😆